base_sdm845_defconfig

```diff
@@ -550,3 +550,9 @@ CONFIG_SND_SOC_MSM_SDW=y
 CONFIG_SND_SOC_MSM_HDMI_CODEC_RX=y
 CONFIG_SND_SOC_SDM845=y
 CONFIG_SND_SOC_WCD_CPE_SOMC_EXT=y
+CONFIG_CPU_QUIET=y
+CONFIG_CPU_QUIET_MSM=y
+CONFIG_CPU_QUIET_DEFAULT_GOV_RQBALANCE=y
+# Not maintained, requires forward porting to newer kernels
+CONFIG_CPU_QUIET_GOVERNOR_RUNNABLE=n
+
```

Comment:
Regardless of this being merged or not, please don't introduce unnecessary trailing newlines.